plugins/UserCountry/tests/UI/UserCountry_spec.js

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Matomo - free/libre analytics platform
+ *
+ * Screenshot integration tests.
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+describe("UserCountry", function () {
+
+    afterEach(function () {
+        delete testEnvironment.unloadGeoIp2;
+        testEnvironment.save();
+    });
+
+    it('should show geolocation admin without additional providers', async function () {
+        testEnvironment.unloadGeoIp2 = 1;
+        testEnvironment.save();
```

Comment:
minor suggestion: instead of a `unloadGeoIp2` setting, we could have a `pluginsToUnload` setting that could be re-used in other contexts for other plugins. what do you think of that idea?

Reply:
makes sense. change that.